test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -273,6 +273,26 @@ spec setup =
             actual.at "col1" . to_vector . should_equal [1000, 200, 1000, 400, 500, 1000]
             actual.column_names . should_equal ["col0", "col_between", "col1", "def"]
 
+    Test.group prefix+"Table.replace" <|
```

Comment:
Don't this test need to be conditional on replace being imlpemented? it is not in Database so they will fail I think?

Reply:
Done.